test/report/test-report-uv-handles.js

```diff
@@ -2,18 +2,13 @@
 
 // Testcase to check reporting of uv handles.
 const common = require('../common');
+const tmpdir = require('../common/tmpdir');
```

Comment:
There should be a `tmpdir.refresh()` call somewhere in this test to ensure it is empty before use.